zallet/src/config.rs

```diff
@@ -138,6 +142,15 @@ impl BuilderSection {
     }
 }
 
+/// Key store configuration section.
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct KeyStoreSection {
+    /// Path to the age identity file that encrypts key material.
+    // TODO: Change this to `PathBuf` once `age::IdentityFile::from_file` supports it.
+    pub identity: String,
```

Comment:
Why not simply convert from `PathBuf` to `String` at the use site, and have this correct from the outset?

Also do config fields have to be public?

Reply:
Conversion from `PathBuf` to `String` is potentially lossy, as paths may contain invalid UTF-8 on some systems (case in point: Windows).

Abscissa structures its apps as "library + binary that uses it", and AFAICT it usually makes config fields public in the library part so they can be accessed programmatically in tests, but I've not been fully consistent here. We will probably want to do a pass over the entire codebase at some point to normalise / decide what we want to expose in the library view.